tmt/steps/prepare/shell.py

```diff
@@ -8,12 +9,14 @@
 import tmt.steps
 import tmt.steps.prepare
 import tmt.utils
+import tmt.utils.git
 from tmt.container import container, field
 from tmt.steps import safe_filename
 from tmt.steps.provision import Guest
-from tmt.utils import ShellScript
+from tmt.utils import Command, EnvVarValue, ShellScript
 
 PREPARE_WRAPPER_FILENAME = 'tmt-prepare-wrapper.sh'
+env_var = 'TMT_PREPARE_SHELL_URL_REPOSITORY'
```

Comment:
The name should be uppercased, i.e. `ENV_VAR`, as it's a module-level "constant". In any case, I suspect it may become a class-level variable, so we can parameterize it if we decide to have different one in `prepare` and `finish`.